src/base/bittorrent/session.cpp

```diff
@@ -440,6 +443,11 @@ Session::Session(QObject *parent)
     // initialize PortForwarder instance
     Net::PortForwarder::initInstance(m_nativeSession);
 
+#if LIBTORRENT_VERSION_NUM >= 10100
+    initMetrics();
+    m_statsUpdateTime = QDateTime::currentDateTime();
```

Comment:
Monotonic clock is the better one. So, I would prefer `std::chrono::steady_clock`, but `QElapsedTimer` is fine with me too.